scripts/lighthouse/pages.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = {
+  HOME: '/',
```

Comment:
Can we import these from `routes.js` instead of refedining them?

Reply:
Sadly not as its in react code and this script cannot parse it, tried that already and wanted to use that but couldn't :(

Reply:
Really. This file is just JS?
https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/constants/routes.js

You might wanna change the script to use `babel-node` instead of `node`. Then you can use `import` syntax and everything

Reply:
Oh yeah sorry misread the file you were referring to